azure-pipelines.yml

```diff
@@ -162,6 +162,50 @@ jobs:
               /p:TeamName=DotNetCore
               /p:OfficialBuildId=$(BUILD.BUILDNUMBER)
             displayName: Sign packages
+    - job: Run_SDL
+      displayName: Run SDL tool
+      variables:
+        - group: SDL_Eng_KeyVault
+      steps:
+        - checkout: self
+          clean: true
+        - powershell: eng\common\build.ps1
+            -configuration Release
+            -restore
+            -prepareMachine
+            -ci
+            /p:OfficialBuildId=$(BUILD.BUILDNUMBER)
+          displayName: Windows Build / Publish
+        - task: NuGetToolInstaller@1
+          displayName: 'Install NuGet.exe'
+        - task: NuGetCommand@2
+          displayName: 'Install Guardian'
+          inputs:
+            restoreSolution: $(Build.SourcesDirectory)\eng\common\sdl\packages.config
+            feedsToUse: config
+            nugetConfigPath: $(Build.SourcesDirectory)\eng\common\sdl\NuGet.config
+            externalFeedCredentials: GuardianConnect
+            restoreDirectory: $(Build.SourcesDirectory)\.packages
+        # Script to execute SDL tools. TsaNotificationEmail will be changed to dnceng@ to check what the noise level would be with a regular SDL runs.
```

Comment:
Consider splitting this comment in multiple lines, this line is too long.

Reply:
Done